docs/UserGuide.md

```diff
@@ -283,210 +341,357 @@ Format: `edit INDEX [date/NAME:YYYY-MM-DD[:monthly|:yearly]]…​`
-* `help add` displays a concise guide on how to use the `add` command.
-* `help edit` displays a concise guide on how to use the `edit` command.
-
-  ![result for 'help'](images/HelpAdd.png)
-
-<br>
+<div markdown="block" class="alert alert-success">
+**:heavy_check_mark: Example:** `help add` 
 
-### Remove all data: `clear`
-Removes all contacts in the app.
+Displays a concise guide on how to use the `add` command.
+![result for 'help'](images/HelpAdd.png)
+</div>
 
-Format: `clear`
+<div markdown="block" class="alert alert-success">
+**:heavy_check_mark: Example:** `help edit` 
 
-<div markdown="span" class="alert alert-warning">:exclamation: **Caution:**
-Please be mindful that entering this command will wipe out all your contacts stored in SociaLite!
+Displays a concise guide on how to use the `edit` command.
 </div>
 
 <br>
 
-### Exiting the program : `exit`
+### Exit the program : `exit` <a name="exit"></a>
 
 Exits the program.
 
-Format: `exit`
+<div markdown="block" class="alert alert-primary">
+**:mag_right: Format:**
+`exit`
+</div>
 
 <br>
 
-### Saving the data
-
-SociaLite data are saved in the hard disk automatically after any command that changes the data. There is no need to save manually.
+--------------------------------------------------------------------------------------------------------------------
 
-<br>
+## FAQ <a name="faq"></a>
 
-### Editing the data file
+**Q**: How is my data saved?<br>
+**A**: SociaLite data are saved in the hard disk automatically after any command that alters the data. There is no need to save manually.
 
-SociaLite data are saved as a JSON file `[JAR file location]/data/SociaLite.json`. Advanced users are welcome to update data directly by editing that data file.
+**Q**: How do I transfer my data to another Computer?<br>
+**A**: Install the app in the other computer and overwrite the empty data file it creates with the file that contains the data of your previous SociaLite home folder.
 
+**Q**: Can I edit my data?<br>
+**A**: SociaLite data are saved as a JSON file `[JAR file location]/data/SociaLite.json`. Advanced users are welcome to update data directly by editing that data file.
 <div markdown="span" class="alert alert-warning">:exclamation: **Caution:**
 If your changes to the data file makes its format invalid, SociaLite will discard all data and start with an empty data file at the next run.
 </div>
 
 --------------------------------------------------------------------------------------------------------------------
 
-## FAQ <a name="faq"></a>
-
-**Q**: How do I transfer my data to another Computer?<br>
-**A**: Install the app in the other computer and overwrite the empty data file it creates with the file that contains the data of your previous SociaLite home folder.
-
---------------------------------------------------------------------------------------------------------------------
-
-## Command summary <a name="summary"></a>
+## Command Summary <a name="summary"></a>
```

Comment:
For this section I have 2 small feedbacks!
* Should we split it to 3 columns, first being command action, second being format and third being example (if any)?
* For better navigability, would it be a good idea to do rename this section into "Command Index" and arrange all the possible commands alphabetically haha

Reply:
Great advice! I'll incorporate both suggestions! 👍

Reply:
Section has been renamed to Command Index and arranged according to alphabetical order in latest commit. Naming of actions has also been standardized!